Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM yandex/clickhouse-client:19.5.3.8
+
+RUN apt-get update && apt-get install -y curl python python-pip && pip install elasticsearch-loader==0.2.25
+RUN curl https://sdk.cloud.google.com | bash && echo "source /root/google-cloud-sdk/completion.bash.inc; source /root/google-cloud-sdk/path.bash.inc" > /root/.bashrc
+COPY . ./genetics-backend
+# Fine-tuned clickhouse settings to enable loading of the data.
+# These settings only apply for the client session and do not affect server settings for other connections (e.g. web application).
+COPY ./clickhouse_client_settings.xml /etc/clickhouse-client/conf.d/
```

Comment:
I think it is better to have this as a volume in docker run

Reply:
You need to have these settings always on to be able to load data successfully. That's why I think it's important to have the settings inside the image. But you could always override it by binding a file during the run if you need to fine-tune it differently.

Reply:
ok :+1: